src/connector.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "connector.h"
+
+#include <proton/connection.h>
+
+#include "connection.h"
+#include "server_private.h"
+
+ALLOC_DEFINE(qd_connector_t);
+
+const char* qd_connector_policy_vhost(qd_connector_t* connector) {
```

Comment:
Our code convention puts the opening brace for functions on its own line directly under the function.  All other C constructs - structs, enums, control statements - put the opening brace on the same line.

Closing braces always appear on their own line.